### ISSUE TITLE
Specify longitude and language via command line

### ISSUE THREAD

**Issue by nicolas-raoul** [nicolas-raoul/wikidata-missing-pictures-offline#1]
should not be hard

**Comment by nsaiisasidhar** [comment]
Hi, may I know if it is longitude & format (i.e., GPX, KML, KMZ). Also is it required that longitude should be two input vals (like in `seq val1 0.1 val2`).

**Comment by nicolas-raoul** [comment]
@nsaiisasidhar 
You can modify the scripts to get the points between longitude1 and longitude2. The present issue is about specifying them from command line.

Language is about the `\"en\"` part.

The 3 formats GPX/KML/KMZ are always generated. The time taken by generating the 3 is negligible compared to the time taken by the network queries.

**Comment by nsaiisasidhar** [comment]
Hi, can you please let me know if the longitude values entered need to be validated. If so is the below criteria fine.

```
if ![ ! -z $1 -a $1 -ge -180 -a $1 -le 179.8 ]
then MIN_LONGITUDE = $1   
else MIN_LONGITUDE = -180 (For MAX_LONGITUDE = 179.9)
fi
```

**Comment by nicolas-raoul** [comment]
Yes validation would be great, but better exit with an error message, so
that the user can correct, rather than use a default value.

I recently pushed latitudes, so please git pull first 🙂

**Comment by nsaiisasidhar** [comment]
Hi, for the language part, can you please let me know  
1. What are the accepted language values for input, 
2. If all needs to done is to get the **item_label** and **class_label** in that input language IF AVAILABLE, or else in ANY or a DEFAULT language.

**Comment by nsaiisasidhar** [comment]
@nicolas-raoul So, are LATITUDES also need to be included in the input values.

**Comment by nicolas-raoul** [comment]
1: I believe language codes are the first column here: https://www.wikidata.org/wiki/Help:Wikimedia_language_codes/lists/all

2: get the item_label and class_label in that input language IF AVAILABLE, or else in ANY language.

As a bonus, it would be great if the user could specify several languages, for instance: Brazilian, Portuguese, Spanish, English.

**Comment by nicolas-raoul** [comment]
Yes, latitudes should be included as input values.
